cmd/kubelet/app/server.go

```diff
@@ -614,9 +614,20 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, stopCh <-chan
 		kubeDeps.Auth = auth
 	}
 
+	if s.CgroupsPerQOS && s.CgroupRoot == "" {
+		klog.Info("--cgroups-per-qos enabled, but --cgroup-root was not specified.  defaulting to /")
+		s.CgroupRoot = "/"
+	}
+
 	if kubeDeps.CAdvisorInterface == nil {
+		cgroupRoots := []string{
+			s.RuntimeCgroups,
+			s.SystemCgroups,
+			s.KubeletCgroups,
+			s.CgroupRoot,
```

Comment:
So if `!CgroupsPerQOS`, this could be `""`? Is that OK? 

Or should `--cgroup-root` always default to `/`?

Reply:
I decided to leave `s.CgroupRoot` alone, and just fall back to collecting all cgroups if `s.CgroupRoot` isn't specified.